packages/web3-eth-miner/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "web3-eth-miner",
+    "namespace": "ethereum",
+    "version": "1.0.0-beta.52",
+    "description": "Web3 module to interact with the Ethereum blockchain accounts stored in the node.",
+    "repository": "https://github.com/ethereum/web3.js/tree/1.0/packages/web3-eth-miner",
+    "license": "LGPL-3.0",
+    "main": "dist/web3-eth-miner.cjs.js",
+    "module": "dist/web3-eth-miner.esm.js",
+    "browser": "dist/web3-eth-miner.umd.js",
+    "scripts": {
+        "build": "rollup -c",
+        "dev": "rollup -c -w",
+        "test": "jest",
+        "dtslint": "dtslint types --onlyTestTsNext"
+    },
+    "types": "types/index.d.ts",
+    "dependencies": {
```

Comment:
The module eth-accounts isn't used here. I will update the dependencies on all new modules after the merge happened but feel free to do it for me.:)

Reply:
yes sure.